Cargo.toml

```diff
@@ -19,9 +19,11 @@ exclude       = [
 ]
 categories = ["network-programming", "data-structures"]
 
+[features]
+serde_impl = ["serde", "serde_test"]
+
 [dependencies]
 byteorder = "1.0.0"
 iovec = "0.1"
-
-[dev-dependencies]
-tokio-core = "0.1.0"
+serde = { version = "1.0", optional = true }
+serde_test = { version = "1.0", optional = true }
```

Comment:
`serde_test` should be a dev-dependency. That way you can call the serde feature simply "serde" instead of "serde_impl".